accurest-core/src/main/groovy/io/codearte/accurest/builder/JaxRsClientSpockMethodBodyBuilder.groovy

```diff
@@ -15,6 +16,9 @@ class JaxRsClientSpockMethodBodyBuilder extends SpockMethodBodyBuilder {
 	}
 
 	@Override
+	protected void given(BlockBuilder bb) {}
```

Comment:
In this class it's same, but they they override two different methods with two different implementations in the base class. Basically, I want to suppress the base class code for JaxRs.